terminologies/technique/imageAnalysis.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/imageAnalysis",

```

Comment:
for my taste this is too general

Reply:
Deleted it from PanProject list. I checked KG and it was used in 10+ datasets. These need to be check case-by-case anyway and change to something more specific.